image_rect_topic: "/camera"
cloud_projected_topic: "/cloud_painter/cloud_projected"
cloud_filtered_topic: "/cloud_filter/filtered_clouds"
# cloud_filtered_topic: "/cloud_painter/filtered_clouds"

yolo_topic: "/yolov5/BoundingBoxes"